imports/client/components/CallSection.tsx

```diff
@@ -54,13 +54,20 @@ const CallStateIcon = styled.span`
 const MutedIcon = styled(CallStateIcon)`
   top: 0;
   border-bottom-left-radius: 6px;
+  border: 0.5px solid #0d6efd;
+  border-left: 0.5px solid #bbb;
+  border-bottom: 1px solid #bbb;
 `;
 
 const DeafenedIcon = styled(CallStateIcon)`
   bottom: 0;
   border-top-left-radius: 6px;
-  font-size: 8px;
+  border: 0.5px solid #0d6efd;
+  border-left: 0.5px solid #bbb;
+  border-top: 1px solid #bbb;
   text-align: right;
+  color: black;
+  font-size: 9px;
 `;
 
 const RemoteMuteButton = styled.div`
```

Comment:
It's not in the diff, so I can't give you the inline fix, but a few lines below this, you also need to increase the `width` & `height` of `RemoteMuteButton` from 40px to 44px